.bashrc

```diff
@@ -0,0 +1,12 @@
+# git add and commit
```

Comment:
Not sure that you meant for this file to be part of this repo?

Reply:
I was testing shortcuts that Todd created. This allows me use gacp to add, commit and push all at once